Modified k8s readme to show API logs (#660)
dapr/quickstarts: tutorials/hello-kubernetes/README.md

## Step 3 - Deploy the Node.js app with the Dapr sidecar

> `dapr.io/enable-api-logging: "true"` annotation is added to node.yaml file by default to see the API logs.

Reviewer: This sentence
"dapr.io/enable-api-logging: "true" annotation is added to node.yaml file by default to see the API logs."

should be after this sentence
"dapr.io/app-id: nodeapp - this assigns a unique id or name to the Dapr application, so it can be sent messages to and communicated with by other Dapr apps."

Successfully persisted state
```

## Step 7 - Observe API call logs

Now that the Node.js and Python applications are deployed, watch API call logs come through:

> Get the API call logs of the Node.js app:

Reviewer: ```suggestion
Get the API call logs of the node app:
```

<!-- STEP
expected_stdout_lines:
  - 'level=info msg="HTTP API Called: POST /v1.0/state/statestore"'
expected_stderr_lines:
output_match_mode: substring
name: Read nodeapp logs
-->

```bash

> POD=$(kubectl get pod -l app=node -o jsonpath="{.items[0].metadata.name}")

Reviewer: ```suggestion
POD=$(kubectl get pod -l app=node -o jsonpath="{.items[0].metadata.name}")
kubectl logs $POD daprd
```
kubectl logs --selector=app=node -c daprd --tail=-1

kubectl logs $POD daprd
```

<!-- END_STEP -->

> If API logs are logged, you should see logs like this:

Reviewer: ```suggestion
When save state API calls are made, you should see logs similar to this:
```

```
time="2022-04-25T22:46:09.82121774Z" level=info msg="HTTP API Called: POST /v1.0/state/statestore" app_id=nodeapp instance=nodeapp-7dd6648dd4-7hpmh scope=dapr.runtime.http-info type=log ver=1.7.2
time="2022-04-25T22:46:10.828764787Z" level=info msg="HTTP API Called: POST /v1.0/state/statestore" app_id=nodeapp instance=nodeapp-7dd6648dd4-7hpmh scope=dapr.runtime.http-info type=log ver=1.7.2

> ```

Reviewer: Also add this

"Get the API call logs of the python app:

kubectl logs --selector=app=python -c daprd --tail=-1
